src/main/java/com/alvarium/contracts/AnnotationType.java

```diff
@@ -1,5 +1,10 @@
 package com.alvarium.contracts;
 
+import com.google.gson.annotations.SerializedName;
+
 public enum AnnotationType {
```

Comment:
The rest of the enum serializations are implemented in the PR #46. Once it's merge, I will rebase and resolve conflicts.